src/sap.ui.core/src/sap/ui/core/Configuration.js

```diff
@@ -786,7 +787,7 @@ sap.ui.define(['jquery.sap.global', 'sap/ui/base/Object', './Locale', 'sap/ui/th
 		},
 
 		/**
-		 * Return whether the controller code is deactivated. During design mode the 
+		 * Return whether the controller code is deactivated. During design mode the
```

Comment:
Huh, sentence abruptly ends - but that's not your mistake. I told the developer who broke this.